### PR TITLE
Ci/semaphore to GitHub actions

### PR DESCRIPTION
**Pull request by PauloGoncalvesBH**
## Pull Request Checklist

- Change CI tool, Semaphore to Github Actions, to provide faster and better feedback and centralize code on repository.
- Remove `pre-commit` dependency.
  - Execute lint on build instead of prettier on pre-commit.

**IMPORTANT:** Unplug Semaphore CI.

- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
- [x] Have you successfully ran tests with your changes locally?
- [ ] Have you written new tests for your changes, if necessary?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
- [ ] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?

> Pull requests that do not pass the automated tests on [SemaphoreCI](http://semaphoreci.com) and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed

### CODE REVIEW

**Comment by PauloGoncalvesBH** [comment]
Add Github Actions build as required on PR and remove semaphoreci.

**Comment by PauloGoncalvesBH** [comment]
Do you think better execute build on `push` only or include `commit`?

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - name: Node.js Setup 
+      uses: actions/setup-node@v1.1.0
+    - name: Installation of dependencies
+      run: npm ci
+    - name: Lint
+      run: npm run lint
+    - name: Unit tests
+      run: npm run test:unit
+    - name: E2E tests
+      run: npm run test:e2e
```
Comment (wlsf82) [inline .github/workflows/continuous-integration.yml]: Should we have an empty line in the end of this file to avoid `No newline at the end of file`?

**Comment by wlsf82** [comment]
I loved this MR, @PauloGoncalvesBH. I have left just a suggestion. As soon as this is implemented I'll go ahead and merge it. Thank you!

**Comment by wlsf82** [comment]
> Do you think better execute build on `push` only or include `commit`?

I'm fine with push only.

**Pull request merged**